node/core/pvf/src/host.rs

```diff
@@ -72,7 +74,7 @@ impl ValidationHost {
 	/// situations this function should return immediately.
 	///
 	/// Returns an error if the request cannot be sent to the validation host, i.e. if it shut down.
-	pub async fn heads_up(&mut self, active_pvfs: Vec<Pvf>) -> Result<(), String> {
+	pub async fn heads_up(&mut self, active_pvfs: Vec<PvfPreimage>) -> Result<(), String> {
```

Comment:
This is the public interface of this crate. As far as I can see, `PvfPreImage` is not exported from this crate and this is undesired (if not making the API unusable)